controllers/itineraries.js

```diff
@@ -65,10 +65,10 @@ module.exports = function (app) {
     });
 
     // DELETE itinerary - yes
-    app.delete('/itineraries/:id', function (req, res) {
```

Comment:
What is the point of changing this if there is no obvious way to delete an itinerary?